src/Documentation/RightPanel/RightPanel.js

```diff
@@ -6,7 +6,7 @@ import { LightButton } from '../LightButton'
 import throttle from 'lodash.throttle'
 
 const ROOT_ELEMENT = 'bodybag'
-const MARKDOWN_ROOT = '.markdown-body'
+const MARKDOWN_ROOT = '#markdown-root'
```

Comment:
if it's not used yet, let's remove it from this PR

Reply:
@shcheklein Used what? Can you please elaborate? :)

Reply:
yep, my bad, I see.